### PR TITLE
Fix deployment command name

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: bindata/v3.11.0/kube-controller-manager/deployment.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
       - name: controller-manager
         image: ${IMAGE}
         imagePullPolicy: IfNotPresent
-        command: ["hypershift", "openshift-kube-controller-manager"]
+        command: ["hyperkube", "kube-controller-manager"]
         args:
         - "--config=/var/run/configmaps/config/config.yaml"
```
Comment (mfojtik) [inline bindata/v3.11.0/kube-controller-manager/deployment.yaml]: --openshift-config

Comment (sttts) [reply bindata/v3.11.0/kube-controller-manager/deployment.yaml]: fixed

**Pull request merged**